geocoder/ip-addresses/sql/build_data_table.sql

```diff
@@ -4,5 +4,5 @@
 -- Clear table
 DELETE FROM ip_address_locations;
 -- Updates table with new source data
-INSERT INTO ip_address_locations (the_geom, network_start_ip) SELECT the_geom, network_start_ip::inet FROM latest_ip_address_locations;
+INSERT INTO ip_address_locations (the_geom, network_start_ip) SELECT the_geom, ('::ffff:' || split_part(network, '/', 1))::inet FROM latest_ip_address_locations;
```

Comment:
I don't agree. IP update process is not automated at all and it involves manual importing of the tables, for example. `build_data_table.sql` has never been ready for updates, and it won't be run now. I fixed it because with the current format at the CSV files it's just broken, but I don't see the value of creating a patch out of it.

Reply:
Fair enough